pkgs/misc/pywal/default.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs ? import <nixpkgs> {} }:
+let
+  py = pkgs.python36Packages;
+in py.buildPythonPackage rec {
+  name = "pywal";
+  version = "2.0.4";
+  src = builtins.fetchTarball {
+    url = "https://github.com/dylanaraps/pywal/archive/${version}.tar.gz";
+  };
+  preCheck = ''
+    mkdir tmp
+    HOME=$PWD/tmp
+  '';
+  postCheck = ''
+    rm -r tmp
+  '';
+  propagatedBuildInputs = with pkgs; [ imagemagick feh ] ++ (with py; [ setuptools ]);
```

Comment:
Please also add `meta` attributes.

Reply:
if the program uses `imagemagick` or `feh` executables, adding those to `propagatedBuildInputs` will not work except inside a `nix-shell`. Use `makeWrapper` here instead to extend `PATH`: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/build-managers/sbt-extras/default.nix#L29